frmts/georaster/georaster_dataset.cpp

```diff
@@ -1252,6 +1267,103 @@ GDALDataset *GeoRasterDataset::Create(const char *pszFilename, int nXSize,
         poGRD->poGeoRaster->nPyramidLevels = atoi(pszFetched);
     }
 
+    pszFetched = CSLFetchNameValue(papszOptions, "GENSTATS");
+
+    if (pszFetched != nullptr)
+    {
+        poGRD->poGeoRaster->bGenStats = EQUAL(pszFetched, "TRUE");
+    }
```

Comment:
can be simplified as:
```suggestion
    poGRD->poGeoRaster->bGenStats = CPLFetchBool(papszOptions, "GENSTATS", FALSE);
```
similar occurrences below